lib/client-assets.php

```diff
@@ -369,6 +376,7 @@ function gutenberg_register_scripts_and_styles() {
 		array(
 			'wp-autop',
 			'wp-blob',
+			'wp-block-serialization-default-parser',
 			'wp-block-serialization-spec-parser',
```

Comment:
I think we no longer need to list `wp-block-serialization-spec-parser` as a dependency. In addition, we should stop registering it, too.

Reply:
agreed on this one but I wasn't entirely sure how we wanted this to work…

do we want Gutenberg to automatically replace the spec parser with the "default" one at boot through a filter or do we want the "default" to be the default?

I want the auto-generated parser to be available still, especially for things like diagnostics and exploration.

Reply:
As commented above, it all depends on the way you want to use it. I don't have any strong opinions about it. We should just ensure we don't ship unused code to the end users.

Reply:
Do we have a decision here?

Reply:
I left the spec parser registered but un-enqueued it in 66455b461